services/ui-src/src/components/export/ExportedReportMetadataTable.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { render, screen } from "@testing-library/react";
+// components
+import { ReportContext, ExportedReportMetadataTable } from "components";
+// utils
+import { ReportShape, ReportStatus, ReportType } from "types";
+import wpVerbiage from "verbiage/pages/wp/wp-export";
+import sarVerbiage from "verbiage/pages/sar/sar-export";
+import { bodyRowContent, headerRowLabels } from "./ExportedReportMetadataTable";
+
+const mockWPContext = {
+  report: {
+    submissionName: "Mock submit",
+    dueDate: 1712505600000,
+    lastAltered: 1712305600000,
+    status: ReportStatus.SUBMITTED,
+    lastAlteredBy: "Mock editor",
+  },
+  reportType: ReportType.WP,
+  verbiage: wpVerbiage.reportPage,
+};
+
+const mockSARContext = {
+  report: {
+    submissionName: "Mock submit",
+    dueDate: 1712505600000,
+    lastAltered: 1712305600000,
+    status: ReportStatus.SUBMITTED,
+    lastAlteredBy: "Mock editor",
+  },
+  reportType: ReportType.SAR,
+  verbiage: sarVerbiage.reportPage,
+};
+
+const metadataTableWithContext = (context: any) => {
+  return (
+    <ReportContext.Provider value={context}>
+      <ExportedReportMetadataTable
+        reportType={context.reportType}
+        verbiage={context.verbiage}
+      />
+    </ReportContext.Provider>
+  );
+};
+
+describe("ExportedReportMetadataTable renders", () => {
+  it("Should render visibly", () => {
+    const { getByTestId } = render(metadataTableWithContext(mockWPContext));
+    const metadataTable = getByTestId("exportedReportMetadataTable");
+    expect(metadataTable).toBeVisible();
+  });
+});
+
+describe("ExportedReportMetadataTable displays the correct content", () => {
+  it("Should show the correct headers for WP", () => {
+    render(metadataTableWithContext(mockWPContext));
+    const headerTexts = Object.values(
+      wpVerbiage.reportPage.metadataTableHeaders
+    );
+    for (let headerText of headerTexts) {
+      const headerCell = screen.getByText(headerText);
+      expect(headerCell).toBeVisible();
+    }
+  });
+  it("Should show the correct headers for SAR", () => {
+    render(metadataTableWithContext(mockSARContext));
+    const headerTexts = Object.values(
+      sarVerbiage.reportPage.metadataTableHeaders
+    );
+    for (let headerText of headerTexts) {
+      const headerCell = screen.getByText(headerText);
+      expect(headerCell).toBeVisible();
+    }
+  });
+  /**
+   *   it("Should show the correct data for WP", () => {
+   *     render(metadataTableWithContext(mockWPContext));
+   *     const cellTexts = ["Mock submit", "04/07/2024", "04/05/2024", "Submitted", "Mock editor"];
+   *     for (let cellText of cellTexts) {
+   *       const cell = screen.getByText(cellText);
+   *       expect(cell).toBeVisible();
+   *     }
+   *   });
+   */
```

Comment:
fix or remove

Reply:
ack! I forgot about that one! I might try to fix it if i can pair w someone